src/NuGet.Core/NuGet.Configuration/Utility/PathValidator.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Text.RegularExpressions;
+using NuGet.Common;
+
+namespace NuGet.Configuration
+{
+    public static class PathValidator
```

Comment:
:+1: I agree, NuGet.Common would be best, this seems like a very common operation.